environment.yml

```diff
@@ -20,6 +20,7 @@ dependencies:
   - pytorch3d
   - pip:
     - geomstats
+    - git+https://github.com/compSPI/compSPI.git
     - git+https://github.com/compSPI/ioSPI.git
```

Comment:
ioSPI not needed after https://github.com/compSPI/simSPI/pull/66 . So once that is merged into master, you can update.